braintumorclassification/frontend/app.py

```diff
@@ -5,20 +5,72 @@
 from io import StringIO
 from io import BytesIO
 from PIL import Image
+from PIL import ImageOps
+import cv2
+import tensorflow as tf
+import os
+from matplotlib.pyplot import imread
+import matplotlib.pyplot as plt
+import cv2
+from scipy import stats
+from tensorflow.keras.utils import to_categorical
+#from tqdm import tqdm
+from PIL import Image
+import tensorflow as tf
+from tensorflow.keras.utils import image_dataset_from_directory
+from tensorflow.keras.models import Sequential, Model
+from tensorflow.keras import layers, optimizers, callbacks
+from tensorflow.keras.preprocessing.image import array_to_img, img_to_array, load_img
+from tensorflow.keras.applications.vgg16 import VGG16, preprocess_input
+from tensorflow.keras.models import load_model
+from PIL import Image
+import requests
+from io import BytesIO
+
+
+st.title("Brain Tumor MRI Classification Example")
 
+st.text("Upload a brain MRI Image for image classification as tumor or no-tumor")
 
-st.markdown('''
-First view of our app!
-''')
 
+from img_classification import teachable_machine_classification
+
+uploaded_file = st.file_uploader("Choose a brain MRI ...", type="jpg")
 
 
-uploaded_file = st.file_uploader("Choose a file", type=['jpg', 'jpeg'])
 if uploaded_file is not None:
-    # To convert to a bytes IO:
-    bytes_io = BytesIO(uploaded_file.getvalue())
-    st.write(bytes_io)
 
-    # Can be used wherever a "file-like" object is accepted:
-    img = Image.open(bytes_io)
-    st.image(img)
+        image = load_img(
+            uploaded_file,
+            grayscale=False,
+            color_mode='rgb',
+            target_size=(255, 255),
+            interpolation='nearest',
+            keep_aspect_ratio=False)
+
+        st.image(image, caption='Uploaded MRI.', use_column_width=True)
+
+        st.write("")
+
+        st.write("Classifying...")
+
+        model_EfficientNetv2 = '/home/aydogan/code/Victorvone/braintumorclassification/braintumorclassification/frontend/EfficientNetv2.h5'
+
+        label = teachable_machine_classification(image, model_EfficientNetv2)
+
+        if label == 0:
```

Comment:
Instead of "Glioma has been detected" can we write: "Glioma tumor has been detected"?
Same goes for the other types.